django/contrib/postgres/fields/array.py

```diff
@@ -16,6 +18,32 @@
 __all__ = ['ArrayField']
 
 
+class ArrayLiteral:
+    """
+    An object which psycopg2 serializes into an array, but is hashable and
+    comparable like a tuple.
+    """
+
+    def __init__(self, value):
+        self.value = tuple(value)
+
+    def __conform__(self, protocol):
+        if protocol is psycopg2.extensions.ISQLQuote:
```

Comment:
According to the coverage report, this always evaluates to True.

Reply:
I implemented it as described in psycopg2 docs http://initd.org/psycopg/docs/advanced.html?highlight=getquoted#adapting-new-python-types-to-sql-syntax , but it's not clear anybody else has used the idea behind `__conform__`. I'll remove the check on `protocol`.